src/Interfaces/AbstractCentrallySymmetric.jl

```diff
@@ -26,6 +26,7 @@ julia> subtypes(AbstractCentrallySymmetric)
 """
 abstract type AbstractCentrallySymmetric{N<:Real} <: LazySet{N} end
 
+isconvextype(::Type{<:AbstractCentrallySymmetric}) = false
```

Comment:
This requires some discussion. Currently all subtypes of `AbstractCentrallySymmetric` are convex. Clearly, the name does not require convexity. But that is also true for `AbstractPolyhedron` etc., so the question is: Do we want to keep `AbstractPolyhedron` for convex polyhedra? Do we want to rename the interfaces?

Reply:
I had in mind the star sets, which are centrally symmetric but are not convex. There is an open issue: https://github.com/JuliaReach/LazySets.jl/issues/1357

I didn't propose to rename the interfaces. But if you feel like this needs further discussion we can follow up next thursday on the telco. In this case, i would let `AbstractCentrallySymmetric` outside this PR and let it for follow up work.

Reply:
> In this case, i would let AbstractCentrallySymmetric outside this PR and let it for follow up work.

:+1:

Reply:
By the way, the answer `false` for interfaces does not make sense to me. If we do that, then the default method for `LazySet` should return `false` instead.

Reply:
> By the way, the answer false for interfaces does not make sense to me. 

Why? It means that the interface type does not guarantee that all subtypes are convex.

>  If we do that, then the default method for LazySet should return false instead.

Yes.

Reply:
I left `isconvextype` for a general LazySet to gives an error, because it is the higher supertype and it's used as a fallback. But i can also code it to return `false` instead of an error. Thay may be even more consistent as `isconvextype(MinkowsiSum)` would now give `false` instead of giving  the error `ERROR: `isconvextype` is not implemented for type MinkowskiSum`, which is the behavior in this branch.

Reply:
> But i can also code it to return false instead of an error. Thay may be even more consistent

I totally agree.

Reply:
Ok, done.

Reply:
And for `AbstractCentrallySymmetric` you want to keep it?

Reply:
No, i propose to do https://github.com/JuliaReach/LazySets.jl/issues/1844. Following this thread we would discuss this in the future, but i can add it here as well.

Reply:
Yes, I suggest to do it here. Why introduce something and change it immediately?
Actually, now you can just remove the method.